modules/ROOT/nav.adoc

```diff
@@ -380,14 +380,11 @@
 ** xref:tinymce-and-csp.adoc[Content Security Policies (CSP)]
 ** xref:tinymce-and-cors.adoc[Cross-Origin Resource Sharing (CORS)]
 * Release information
-** xref:release-notes.adoc[Release notes for TinyMCE 6]
+** xref:6.0-release-notes.adoc[Release notes for TinyMCE 6]
```

Comment:
This shouldn't have been changed, this is meant to be an index page to list all releases for TinyMCE 6.